packages/playground/README.md

```diff
@@ -51,3 +51,48 @@ You can also launch `/playground/windows/playground.sln` solution file in Visual
 5. F5! This will build your playground app, and deploy to the target machine.
 6. On the target, press `Load`. This will communicate back to the dev machine bundler and launch the web debugger on the dev machine, and serve the pages back to the app running in the target.
    You need an updated version of the bundler in order for remote debugging to work properly: [PR here](https://github.com/react-native-community/cli/pull/829)
+
+# Playground-win32
+
+Playground-win32 is a sample of an RNW application using XAML Islands that allows testing of various JS files, including RNTester.
+
+## Launching the app
+
+1. Make sure your development machine has been set up with all the system requirements and dependencies mentioned [here](https://microsoft.github.io/react-native-windows/docs/next/rnw-dependencies). Make sure a browser is launched and running.
+
+1. Install Packages with-in the repo
+
+`yarn`
+
+1. Ensure packages are built
+
+`yarn build`
+
+1. Navigate to the `playground` folder
+
+`cd packages\playground`
+
+1. Start Metro
+
+`yarn start`
+
+1. Open `windows\playground-win32.sln` in Visual Studio
+
+1. Set the build configuration to "Debug" and Platform to "x86" or "x64"
+
+1. Hit "Local Windows Debugger"
+
+This will build and deploy the application and the dev tools.
```

Comment:
@rectified95 Instruction vary here.

Reply:
Are these steps necessary, though? I've always been able to just run this solution file as is; selecting the build architecture is not specific to win32.sln

Reply:
Solution defaults to ARM when it opens, so I figured might as well specify for someone who isn't familiar with the fact that we only technically support x86 and x64. Steps for regular playground on this page list out running  from the CLI via `npx react-native run-windows` which won't work for playground-win32 which is why I have the steps listed here on running from VS.

Reply:
Hmm, I didn't realize we removed the VS steps as the default for Playground. Still, the process is the same for Playground and Playground32. I would say add/rearrange info about running from VS to what we have (currently this is only under the 'remote debug' section, but you don't need to be remote debugging to want to use VS app launch.)

Reply:
Sure I can do that.